Classes/ScrnBenelliAttachment.uc

```diff
@@ -0,0 +1,7 @@
+class ScrnBenelliAttachment extends BenelliAttachment;
+
+defaultproperties
+{
+    MeshRef="KF_Weapons3rd3_Trip.Benelli_3rd"
+    Mesh=SkeletalMesh'KF_Weapons3rd3_Trip.Benelli_3rd' //added to fix
```

Comment:
Explicitly setting the mesh will always load the skeletal mesh together with ScrnBalance. And we don't want this. We want a mesh to be loaded only when ScrnBenelliAttachment is spawned. So we need to use MeshRef and load Mesh inside `PreloadAssets`. I'll check the code why the MeshRef doesn't work in the first place.

Reply:
I looked at the original code, BenelliAttachment has the correct MeshRef line as it should. However the ShotgunAttachment class it extends from has Mesh set to Mesh=SkeletalMesh'KF_Weapons3rd_Trip.Shotgun_3rd' which is the cause of this bug (thanks tripwire), I added the Mesh line make it like a working copy of BenelliAttachment, but after learning about this behaviour it would be better to just make a whole new ScrnBenelliAttachment that extends KFWeaponAttachment.